app/models/team-payment-method.ts

```diff
@@ -0,0 +1,9 @@
+import Model, { attr, belongsTo } from '@ember-data/model';
+import type TeamModel from './team';
+import type UserModel from './user';
+
+export default class TeamPaymentMethod extends Model {
+  @attr('date') declare createdAt: Date;
```

Comment:
Switch order with relationships (these models must be old btw - in the new TS ones I think we're following this rule properly)